lib/shiftable.rb

```diff
@@ -1,4 +1,4 @@
-module Crackable
+module Shiftable
```

Comment:
I changed the name from Crackable to Shiftable because Shiftable is more descriptive of the behaviors in the module.